xeth/xeth.go

```diff
@@ -608,6 +608,12 @@ func (self *XEth) Call(fromStr, toStr, valueStr, gasStr, gasPriceStr, dataStr st
 	return common.ToHex(res), err
 }
 
+func (self *XEth) ConfirmTransaction(tx string) bool {
```

Comment:
Because natspec.js needs the whole JSON-encoded RPC request, so it was easier to pass it than to re-encode.